src/lib.rs

```diff
@@ -941,7 +964,204 @@ impl UInt {
+            // which comfortably fits in a u64.
+            None => unsafe { unreachable_unchecked() },
+        })
+    }
+
+    /// Wrapping (modular) subtraction. Computes self - rhs, wrapping around at the boundary of the type.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// # use js_int::UInt;
+    /// assert_eq!(UInt::from(100u32).wrapping_sub(UInt::from(100u32)), UInt::from(0u32));
+    /// assert_eq!(UInt::from(100u32).wrapping_sub(UInt::from(UInt::MAX)), UInt::from(101u32));
+    /// ```
+    #[must_use]
+    pub fn wrapping_sub(self, rhs: Self) -> Self {
+        Self::new_wrapping(self.0.wrapping_sub(rhs.0))
+    }
+
+    /// Wrapping (modular) multiplication. Computes self * rhs, wrapping around at the boundary of the type.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// # use js_int::UInt;
+    /// assert_eq!(UInt::from(12u32).wrapping_mul(UInt::from(10u32)), UInt::from(120u32));
+    /// assert_eq!(UInt::from(2u32).wrapping_mul(UInt::MAX), UInt::MAX - UInt::from(1u32));
+    #[must_use]
+    pub fn wrapping_mul(self, rhs: Self) -> Self {
+        Self::new_wrapping(self.0.wrapping_mul(rhs.0))
+    }
+
+    /// Wrapping (modular) division. Computes `self / rhs`. Wrapped division on unsigned types is
+    /// just normal division. There's no way wrapping could ever happen. This function exists, so
+    /// that all operations are accounted for in the wrapping operations.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// # use js_int::UInt;
+    /// assert_eq!(UInt::from(100u32).wrapping_div(UInt::from(10u32)), UInt::from(10u32));
+    /// ```
+    #[must_use]
+    pub fn wrapping_div(self, rhs: Self) -> Self {
+        Self(self.0.wrapping_div(rhs.0))
+    }
+
+    /// Wrapping (modular) remainder. Computes `self % rhs`. Wrapped remainder calculation on
+    /// unsigned types is just the regular remainder calculation. There's no way wrapping could ever
+    /// happen. This function exists, so that all operations are accounted for in the wrapping
+    /// operations.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// # use js_int::UInt;
+    /// assert_eq!(UInt::from(100u32).wrapping_rem(UInt::from(10u32)), UInt::from(0u32));
+    /// ```
+    #[must_use]
+    pub fn wrapping_rem(self, rhs: Self) -> Self {
+        Self::new_wrapping(self.0.wrapping_rem(rhs.0))
+    }
+
+    /// Wrapping (modular) exponentiation. Computes `self.pow(rhs)`, Wrapping around the boundary
+    /// of the type.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// # use js_int::UInt;
+    /// assert_eq!(UInt::from(100u32).wrapping_rem(UInt::from(10u32)), UInt::from(0u32));
+    /// ```
+    #[must_use]
+    pub fn wrapping_pow(self, exp: u32) -> Self {
+        Self::new_wrapping(self.0.wrapping_pow(exp))
+    }
+
+    /// Calculates `self` + `rhs`
```

Comment:
Everything else has a dot at the end of the first paragraph, right?

```suggestion
    /// Calculates `self` + `rhs`.
```